src/dom-to-image.js

```diff
@@ -6,6 +6,14 @@
     var fontFaces = newFontFaces();
     var images = newImages();
 
+    // Default impl options
+    var defaultOptions = {
+        // Default placeholder is a 1x1 gray square
+        placeholder: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAMSURBVBhXY7h79y4ABTICmGnXPbMAAAAASUVORK5CYII=",
+        // Default cache bust is true
+        cacheBust: true
```

Comment:
I was thinking about it and I would still set this to `false` by default. If I understand the whole thing correctly, this option causes re-downloading the images and fonts each time the thing is executed. That would slow down the rendering for every user of the lib regardless whether they face the problem you're facing